### PR TITLE
Adding reagent validation results from multiple contributors.

### PR DESCRIPTION
**Pull request by zivy**

### CODE REVIEW

File: docs/supporting_material/CXCR3_AF700/Human_Tonsil_FFPE_CXCR3_FAB160N.jpg
Comment (radtkea) [inline docs/supporting_material/CXCR3_AF700/Human_Tonsil_FFPE_CXCR3_FAB160N.jpg:1]: It appears that this image is not included in the markdown file for CXCR3 catalog number FAB160N.

File: docs/supporting_material/Rabbit_IgG__H+L__AF700/0000-0003-4379-8967.md
Comment (radtkea) [inline docs/supporting_material/Rabbit_IgG__H+L__AF700/0000-0003-4379-8967.md:1]: No image is associated with this entry. Please add to markdown: Human_jejunum_CD3_BV510_300448_CollIV_ab6586_AF700_A-21038_EpCAM_AF594_324228.jpg

**Review by radtkea — COMMENTED** [review]
Thank you very much! There are a few entries missing their supporting image. Please see my comments.

File: docs/supporting_material/CXCR3_AF700/Human_Tonsil_FFPE_CXCR3_FAB160N.jpg
Comment (zivy) [reply docs/supporting_material/CXCR3_AF700/Human_Tonsil_FFPE_CXCR3_FAB160N.jpg:1]: Added.

File: docs/supporting_material/Mouse_IgG__H+L__AF555/0009-0004-2746-8743.md
[the remark this answers is not in the document]
Comment (zivy) [reply docs/supporting_material/Mouse_IgG__H+L__AF555/0009-0004-2746-8743.md:1]: Done.

File: docs/supporting_material/Rabbit_IgG__H+L__AF700/0000-0003-4379-8967.md
Comment (zivy) [reply docs/supporting_material/Rabbit_IgG__H+L__AF700/0000-0003-4379-8967.md:1]: Done.

**Review by radtkea — APPROVED** [review]
Thanks for adding the images!

**Pull request merged**